Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

```diff
@@ -20,10 +20,11 @@ import androidx.annotation.FloatRange
 import androidx.annotation.StringRes
 import androidx.compose.animation.AnimatedFloatModel
 import androidx.compose.animation.Crossfade
-import androidx.compose.animation.animate
+import androidx.compose.animation.animateAsState
 import androidx.compose.animation.animatedFloat
 import androidx.compose.animation.core.AnimationSpec
 import androidx.compose.animation.core.SpringSpec
+import androidx.compose.animation.core.animateAsState
```

Comment:
Two `animateAsState` imports here. Which one to use? I assume it's the one at the top

Reply:
Both are needed, the first is for animating a `Float` and the later is for animating a `color`. Does seem strange that they're in different packages, will investigate.